assets/lib/font-awesome/update-fontawesome.php

```diff
@@ -0,0 +1,58 @@
+<?php
+// How to update a version for Font Awesome:
+// FYI: "js" and "json" folders belongs to elementor and not fontawesome.
+
+// 1. Download the new version of Font Awesome.
+// 2. Replace the fontawesome folder excluding this file and migration folder.
+// 3. run this script.
+// 4. Files in js and json folders should be updated with the new icons.
+// 5. Delete redundant files in order to keep Elementor plugin size small.
+
+$metadata_path = 'metadata/icons.json';
```

Comment:
Please use absolute paths here and next
```suggestion
$metadata_path = __DIR__ . '/metadata/icons.json';
```